frontend/static/manifest.webmanifest

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "PocketID",
```

Comment:
Why did you revert back to a static file? As per my comment, we would need the `name` to come from the app's name, which is dynamic.